src/testdrive/src/parser.rs

```diff
@@ -162,6 +174,53 @@ pub fn validate_ident(name: &str) -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+fn parse_version_check(line_reader: &mut LineReader) -> Result<(Command, String), PosError> {
```

Comment:
Shall we add a unit test for this logic? Not sure myself.

Reply:
I'm also not sure, there are no unit tests for testdrive so far, so my guess is that since it's a tool for testing only, not so important to test it.